tests/models/clip/test_modeling_clip.py

```diff
@@ -189,114 +193,17 @@ def test_model(self):
         config_and_inputs = self.model_tester.prepare_config_and_inputs()
         self.model_tester.create_and_check_model(*config_and_inputs)
 
-    def test_attention_outputs(self):
```

Comment:
Why is this test deleted ?

Reply:
Because it's not required to overwrite this. By simply setting the `seq_length` attribute of the `ModelTester`, this test passes without requiring to overwrite it (same with `test_hidden_states_output`).